.circleci/config.yml

```diff
@@ -532,36 +571,58 @@ workflows:
   compile:
     jobs:
       - pre_deps
-      - build_db_migrate_image
+      - build_db_migrate_image:
+          filters:
+            branches:
+              ignore:
+                - jb-clickable-prototype
```

Comment:
Let's use `scylla-prototype` as a branch since presumedly I won't be the only dev before long. I renamed the branch on GH just now.